### PR TITLE
Simplify templates

### PR DESCRIPTION
**Pull request by FinalAngel**
The goal of this Pull Request is:

- Simplify template structure
- Remove unnecessary placeholder and blocks
- In general make everything simple for easier explanation

We also want to remove ``render_placeholder view.config`` and later only mention it in the documentation (separate PR will follow).

### CODE REVIEW

File: aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/includes/event.html [hunk 1]
```diff
@@ -4,73 +4,69 @@
...
+        {% else %}
+            {{ event.title }}
+        {% endif %}
+    </h2>
+
+    <div class="aldryn-events-meta">
+        {% if event.location %}
+            <p class="location">
+                {% if event.location_lat and event.location_lng %}
+                <a href="http://maps.google.com?q={{ event.location_lat }},{{ event.location_lng }}" target="_blank">
```
Comment (lory87) [inline aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/includes/event.html]: indent

File: aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/includes/event.html [hunk 2]
```diff
@@ -4,73 +4,69 @@
...
+                {% else %}
+                <a href="http://maps.google.com?q={{ event.location|slugify }}" target="_blank">
```
Comment (lory87) [inline aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/includes/event.html]: indent

File: aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/includes/event.html [hunk 3]
```diff
@@ -4,73 +4,69 @@
...
+                {% endif %}
+                    {{ event.location }}
```
Comment (lory87) [inline aldryn_events/boilerplates/bootstrap3/templates/aldryn_events/includes/event.html]: indent

File: aldryn_events/templates/aldryn_events/includes/event.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            <a href="{{ event.get_absolute_url }}">{{ event.title }}</a>
+        {% else %}
+            {{ event.title }}
+        {% endif %}
+    </h2>
+
+    {% if event.location %}
+        <p>
+            {% if event.location_lat and event.location_lng %}
+            <a href="http://maps.google.com?q={{ event.location_lat }},{{ event.location_lng }}" target="_blank">
```
Comment (lory87) [inline aldryn_events/templates/aldryn_events/includes/event.html]: indent

File: aldryn_events/templates/aldryn_events/includes/event.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            {% else %}
+            <a href="http://maps.google.com?q={{ event.location|slugify }}" target="_blank">
```
Comment (lory87) [inline aldryn_events/templates/aldryn_events/includes/event.html]: indent

File: aldryn_events/templates/aldryn_events/includes/event.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+            {% endif %}
+                {{ event.location }}
```
Comment (lory87) [inline aldryn_events/templates/aldryn_events/includes/event.html]: indent

**Comment by lory87** [comment]
lgtm

**Comment by vxsx** [comment]
yeah :+1:

**Pull request merged**